src/client/src/hooks/useItem.js

```diff
@@ -30,7 +32,7 @@ class ItemClient {
     }
```

Comment:
you need to be handling these errors and not 'swallowing' them. This makes it a silent failure and any code calling this just gets a return of undefined and there's no error to handle